src/pt/lermangaonline/src/eu/kanade/tachiyomi/extension/pt/lermangaonline/LerMangaOnline.kt

```diff
@@ -0,0 +1,130 @@
+
+    override val baseUrl = "https://lermangaonline.com.br"
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 1, 1, TimeUnit.SECONDS)
+        .build()
+
+    private fun String.toChapterName() =
+        trim().replace("\\d{1,2}-\\d{1,2}-\\d{2,4}".toRegex(), "")
+
+    private fun String.toChapterNumber(): Float =
+        "\\d+".toRegex().find(trim())?.value?.toFloat() ?: 0F
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("div.capitulo")!!.text().toChapterName()
+        date_upload = (element.selectFirst("span")?.text() ?: "").toDate()
+        chapter_number = name.toChapterNumber()
+        setUrlWithoutDomain(element.absUrl("href"))
+    }
+
+    override fun chapterListSelector() = "div.capitulos a"
+
+    override fun imageUrlParse(document: Document) = ""
+
+    override fun getMangaUrl(manga: SManga): String = baseUrl + manga.url
+
+    override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
+        title = element.selectFirst("section h3")!!.text()
+        thumbnail_url = element.selectFirst("div.poster img")?.absUrl("src")
+        setUrlWithoutDomain(element.selectFirst("div.poster a")!!.absUrl("href"))
+    }
+
+    override fun latestUpdatesNextPageSelector() = "div.wp-pagenavi [aria-current] + a"
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = "$baseUrl/capitulo/page/$page".toHttpUrl().newBuilder()
+            .build()
+        return GET(url, headers)
+    }
+
+    override fun latestUpdatesSelector() = "div.box-indx section.materias article"
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        val info = document.selectFirst("div.box-flex")
+        title = info!!.selectFirst("div.sinopse a")!!.text()
+        description = info.selectFirst("div.sinopse div:nth-child(2)")?.text()
+        thumbnail_url = info.selectFirst("div.poster img")?.srcAttr()
+        initialized = true
+        genre = document.select("div.categorias-blog a").joinToString(", ") { it.text() }
+        status = SManga.UNKNOWN
+        setUrlWithoutDomain(document.location())
+    }
+
+    override fun pageListParse(document: Document): List<Page> {
+        val elements = document.select("div.images img")
+        return elements.mapIndexed { i, el ->
+            Page(i, document.location(), el.srcAttr())
+        }
+    }
+
+    override fun popularMangaFromElement(element: Element) =
+        throw UnsupportedOperationException()
+
+    override fun popularMangaNextPageSelector() =
+        throw UnsupportedOperationException()
+
+    override fun popularMangaRequest(page: Int) =
+        throw UnsupportedOperationException()
+
+    override fun popularMangaSelector() =
+        throw UnsupportedOperationException()
```

Comment:
You should not stub out `popularManga*` functions! The default view when tapping on an extension is the "Popular" view, and the "Latest" view is only a small button to the right". This will confuse users!

It is perfectly acceptable to use the latest updates page for popular manga, if the page doesn't have a proper page for the latter.